components/ui/Modal.tsx

```diff
@@ -30,14 +30,25 @@ export const Modal: React.FC<ModalProps> = props => {
               />
             ) : null}
             {props.arrowLeft ? (
-              <Icon
-                {...testIDProps('arrowLeft')}
-                name="arrow-left"
-                type="material-community"
-                onPress={props.onDismiss}
-                containerStyle={Theme.Styles.backArrowContainer}
-                color={Theme.Colors.Icon}
-              />
+              I18nManager.isRTL ? (
```

Comment:
Did you check it's impact on other screen like QRConsent, MyBindedVCs?

Reply:
verified on the both the QRConsent and MyBindedVCs screens working as expected